Grbl_Esp32/src/GCode.cpp

```diff
@@ -1301,31 +1370,31 @@ uint8_t gc_execute_line(char* line, uint8_t client) {
     // M0,M1,M2,M30: Perform non-running program flow actions. During a program pause, the buffer may
     // refill and can only be resumed by the cycle start run-time command.
     gc_state.modal.program_flow = gc_block.modal.program_flow;
-    if (gc_state.modal.program_flow) {
+    if (gc_state.modal.program_flow != ProgramFlow::Running) {
```

Comment:
switch ?

Reply:
Okay, I changed this to a switch but the control flow is tricky enough that I would like for you to check it carefully.